### DISCUSSION TITLE
Is it possible to install "api", "rmm" and "mesh" on different servers ?

### DISCUSSION THREAD

**Discussion by pd1drone** [amidaware/tacticalrmm#1518] [Q&A] [answered]
Hi I wanted to clarify if it is possible to separate the "api", "rmm" and "mesh" on different servers but have the same root domain name ? 

Example:
Root Domain: example.com
API Domain: api.example.com
RMM Domain: rmm.example.com
Mesh Domain: mesh.example.com

I want each of the api, rmm, and mesh to be installed in a different server, is this possible ? if it is could I get a link to the documentation on how to set this up ? if not then it is OK I just wanted to clarify if this is possible or not. Thank you

Cheers

**Comment by wh1te909** [comment, marked as answer]
it is possible, but not supported therefore there are no docs for this, you would have to figure it out yourself.

**Reply by pd1drone** [reply]
Thanks for confirming this that this is possible👍 .

Cheers
